### ISSUE TITLE
Dota just cut my Phantom Assassin Arcana Progress in half after Turbo Update

### ISSUE THREAD

**Issue by rNtgg** [ValveSoftware/Dota2-Gameplay#9657]
### Cosmetic Item Name

Manifold Paradox

### Description

We know Dota updated turbo to work with PA and Pudge Arcana since yesterday's update for half progress. That's normal, happened with other arcanas aswell.

BUT on my Arcana I had 22 score from playing normal matches, since it would be impossible to upgrade on turbo before. I know this because when I read the update I checked to see if I was far from style 2 and 3.

Today I played my first match with her (after update) and I see my score was halved also, but it is a bug since I played the other 22 victories on normal matches.
![PAArcanaBug](https://github.com/ValveSoftware/Dota2-Gameplay/assets/106766180/15fb0cc6-fc18-468a-81cc-73d01706cba0)

Please check it.

### Example Match ID

7193366367

### Screenshots

![PAArcanaBug](https://github.com/ValveSoftware/Dota2-Gameplay/assets/106766180/c8c04ef3-2464-4fba-bd66-9bed4f20d957)

**Comment by Aluerie** [comment]
I do not want to assume/speculate anything or undermine your score.  

But other issues in this repository suggest that the count mistakenly doubled after yesterday's update. So,,, what if it's just a fix for it. Are you sure you had 22 score and not 11 **before** yesterday update and not after reading the small-patch news.

**Comment by rNtgg** [comment]
Not sure. Checked right after update rolled out and was 22. Your theory is that 22 was doubled and not 11 halved? Hmm..

**Comment by Aluerie** [comment]
Yes. my theory is that the real number is 11 that got mistakenly doubled to 22. Now they fixed it and you got your 11 + 0.5 back. 

For reference https://github.com/ValveSoftware/Dota2-Gameplay/issues/9622

* or [this person](https://www.reddit.com/r/DotA2/comments/143zw7m/where_can_i_see_how_many_points_i_have_left_to/ ) who has 146 count but style 3 is locked because the real number is 73

**Comment by BenLubar** [comment]
I'd need someone from Valve to confirm to know for sure, but my hypothesis is that what they did was:

1. double the gem progress on every existing arcana
2. add code to the game to divide by 2

it looks like the Steam Community version of the item description formatter hasn't been updated yet.

**Comment by Gajeel92** [comment]
My friend also has the same problem. Original assassinations' score was the same when you look at the PA Arcana at Steam Inventory then reduced to half when accessing via ingame. He already filed a ticket. Hope it will be fixed.

**Comment by Aluerie** [comment]
"Reduced to half" was exactly a bugfix for the count being mistakenly doubled. 

All is correct now.

**Comment by HonestBanni** [comment]
can any one explain me what is assassination. and how its work ?  I have pa arcana but need to understand how unlock 2nd and 3rd style.
